model/task/task.go

```diff
@@ -102,6 +102,11 @@ type Task struct {
 	Activated                bool   `bson:"activated" json:"activated"`
 	ActivatedBy              string `bson:"activated_by" json:"activated_by"`
 	DeactivatedForDependency bool   `bson:"deactivated_for_dependency" json:"deactivated_for_dependency"`
+
+	// TaskDepth indicates how far into stepback this task was activated.
```

Comment:
StepbackDepth indicates how far...

Should we make a note that a task with no stepback has depth=0, and the first stepback has depth=1 to clear possible ambiguity over what is the initial depth for stepback?

Reply:
Sure

Reply:
nit: update TaskDepth -> StepbackDepth